### ISSUE TITLE
printp(x) output is inconsistent when x has values greater & smaller than .001

### ISSUE THREAD

**Issue by mvuorre** [crsh/papaja#313]
Currently, if you run `printp(x)` on `x` that contains values both above and below the threshold of 0.001, you get inconsistent output values, some of which contain an operator and some don't. This is especially problematic if you're e.g. constructing a variable that has p values from several tests that you'd like to report together (in a table or in text).

``` r
library(papaja)

ps <- c(1, .05, .00001, NA)
printp(ps)
#> [1] "> .999" ".050"   "< .001" ""

# Reporting in text can result in two operators
paste0("Test ", 1:4, ": p = ", printp(ps))
#> [1] "Test 1: p = > .999" "Test 2: p = .050"   "Test 3: p = < .001"
#> [4] "Test 4: p = "
# or some without equals sign
paste0("Test ", 1:4, ": p ", printp(ps))
#> [1] "Test 1: p > .999" "Test 2: p .050"   "Test 3: p < .001"
#> [4] "Test 4: p "

# New feature is suggested in my fork's devel branch
# remotes::install_github("mvuorre/papaja@devel")
printp(ps, add_eq = TRUE)
#> [1] "> .999" "= .050" "< .001" ""

# Default is add_eq = FALSE, which gives old behavior
printp(ps)
#> [1] "> .999" ".050"   "< .001" ""
```

<sup>Created on 2019-09-05 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9000)</sup>

See the commit [here](https://github.com/mvuorre/papaja/commit/b2df00389bc896cc816cb198f6b06fefbf8352fb). 

An additional benefit of including those extra two lines of code in `printp()` is that you could remove quite a bit of code that does the same in thing in various `apa_print*` methods (e.g. [here](https://github.com/crsh/papaja/search?q=%21grepl%28%22%3C%7C%3E%22%2C&unscoped_q=%21grepl%28%22%3C%7C%3E%22%2C)) and instead replace them with `printp(p, add_eq = TRUE)`

**Comment by crsh** [comment]
Hi Matti, thanks for the suggestion. That makes sense to me. I think, it would even make sense to add this to `printnum()` since `printp()` is just a shortcut. Addition of an equal symbol could be useful in `printnum()` as well when `zero = FALSE` or `gt1 = FALSE`, right?

@mariusbarth, what do you think about this? This would be a more general alternative to your proposed `add_equals()` in #310, right?

**Comment by mvuorre** [comment]
Great, yes it looks like you'd want to put it in `printnum()`.

**Comment by crsh** [comment]
@mariusbarth, you worked on this right? Could you summarize how you solved this (maybe in reference to the above example)?

**Comment by mariusbarth** [comment]
Sure! On the development branch, I basically implemented what Matti suggested: Both `printp()` and the `printnum.numeric()` method now have an additional argument `add_equals` which defaults to `FALSE`. Therefore, both functions by default show their previous behavior. If, instead, you want a consistent output which is always prepended by either `=, <, >`, then you choose `add_equals = TRUE`. Hope this is exactly what you wished for.
